client/signup/steps/website-content/section-types/site-information-section.tsx

```diff
@@ -46,8 +56,16 @@ export function LogoUploadSection( {
 		dispatch( logoRemoved() );
 	};
 
+	const onFieldChanged = ( e: ChangeEvent< HTMLInputElement > ) => {
+		const {
+			target: { value },
+		} = e;
+		dispatch( updateSearchTerms( value ) );
+		onChangeField && onChangeField( e );
```

Comment:
I think this would be a fancier equivalent 😁 

```suggestion
		onChangeField?.( e );
```

Reply:
Good idea! I have made that change in a few places via https://github.com/Automattic/wp-calypso/pull/74273/commits/ff09ba8ce4d6f8a90183f8ce67f715640a23004d.